nengo_bones/templates/slurm-script.sh.template

```diff
@@ -0,0 +1,76 @@
+{% extends "templates/base_script.sh.template" %}
+
+{#
+
+Usage Instructions
+==================
+
+This script is intended to be used to apply a SLURM wrapper around other NengoBones
+generated CI scripts.
+
+.nengobones.yml parameters:
+
+`slurm_command`: The SLURM command used to create the SLURM job. Use quotations to avoid
+problems with the space character. Examples:
+    - "srun"
+    - "srun -pCI -G1"
+    - "sbatch"
+
+`wrapped_commands`: A list of commands to be wrapped with `slurm_command`. The command
+will be executed as: <slurm_command> <command>
+Example, if <slurm_command> is "srun -pCI", and <command> is "pytest", then the command
+will be executed as "srun -pCI pytest"
+
+`slurm_script`: The CI script to be executed, but using the SLURM-wrapped commands
+instead of the typical non-SLURM invocation.
+
+`output_name`: Desired output name of the generated CI script.
+
+#}
+
+{% block install %}
+    bash .ci/{{ slurm_script }}.sh install || STATUS=1
+{% endblock %}
+
+{% block before_script %}
+    bash .ci/{{ slurm_script }}.sh before_script || STATUS=1
+{% endblock %}
+
+{% block script %}
+    # Enable `expand_aliases` option for bash interpreter
+    shopt -s expand_aliases
+
+    # Create aliases for commands to include SLURM command prefix
+    {% for command in wrapped_commands %}
+    alias {{ command }}="{{ slurm_command }} {{ command }}"
+    echo "Created alias for {{ command }}: "
+    type -a {{ command }}
```

Comment:
I don't think you need a new script template for this, I think you could do this with `pre_commands`, same as with the conda installation. E.g., in your docs script you'd put
````
pre_commands:
    - alias sphinx-build="srun sphinx-build"
```

Reply:
I will test this

Reply:
I can't get this to work with `pre_commands`. I think it's the way the aliases work...